test/custom_class/t/t.py

```diff
@@ -0,0 +1,36 @@
+import torch
+torch.ops.load_library("build/t.cpython-37m-darwin.so")
+# print(torch.ops.my_ops.warp_perspective(torch.randn(32, 32)))
+
+@torch.jit.script
+class Bar:
+    def __init__(self):
+        self.x = 1
+        self.y = 2
+
+    # def __init__(self, x_, y_):
+    #     # type: (int, int) -> None
+    #     self.x = x_
+    #     self.y = y_
+
+    def display(self):
+        print(self.x, self.y)
+
+    def display(self, z):
+        print(self.x, self.y, z)
+
+@torch.jit.script
+def f(x):
+    val = torch._C.Foo(5, 3)
```

Comment:
As adam said, `_C` is a reserved internal namespace so we shouldn't have users put objects there.

Reply:
Agreed - I'll work on figuring out where I can put it instead.